app/assets/stylesheets/content/_forms.sass

```diff
@@ -61,6 +61,22 @@ $form--field-types: (text-field, text-area, select, check-box, radio-button, ran
     &.-compressed
       padding: 10px 20px 0 20px
 
+    &.-danger-zone
+      .form--section-title
+        background-color: $content-form-danger-zone-bg-color
+        color: $content-form-danger-zone-font-color !important
+        padding: 0.5rem !important
+        border-radius: 2px
+        border: 1px solid $content-form-danger-zone-bg-color
+
+      .button.-highlight
+        background: $gray-light
```

Comment:
How about `$content-form-danger-zone-font-color`? This would mean that the unhovered button is exactly the inverted hovered button.

I think a white button does not look too bad on the grey background:

![image](https://cloud.githubusercontent.com/assets/453584/9877702/d7f875ea-5bbe-11e5-8012-fb0c89d19d51.png)

(This proposal is of course again because of themability considerations... you will not need to care whether `$grey-light` and the dz-bg-color are an accessible color fit)